docs/configuration.md

```diff
@@ -838,11 +838,13 @@ For more details, see [#1131](https://github.com/docsifyjs/docsify/issues/1131).
 
 > **Warning** Deprecated. Use the CSS var `--theme-color` in your `<style>` sheet. Example:
 >
-> <style>
->   :root {
->     --theme-color: deeppink;
->   }
-> </style>
+```html
+<style>
+   :root {
+     --theme-color: deeppink;
+   }
+</style>
+```
```

Comment:
Oops, I forgot the code fence. I meant for this to be inside the warning. For example it should be like this:

```md
> **Warning** Deprecated. Use the CSS var `--theme-color` in your `<style>` sheet. Example:
>
> ```html
> <style>
>   :root {
>     --theme-color: deeppink;
>   }
> </style>
> ```
```

and here's what it looks like inside the quote block:

> **Warning** Deprecated. Use the CSS var `--theme-color` in your `<style>` sheet. Example:
>
> ```html
> <style>
>   :root {
>     --theme-color: deeppink;
>   }
> </style>
> ```